internal/pkg/template/templates/workloads/partials/cf/autoscaling.yml

```diff
@@ -80,4 +80,161 @@ AutoScalingPolicyECSServiceAverageMemoryUtilization:
+        PolicyDocument:
+          Version: '2012-10-17'
+          Statement:
+            - Sid: ECS
+              Effect: Allow
+              Action:
+                - ecs:DescribeServices
+              Resource: "*"
+              Condition:
+                ArnEquals:
+                  'ecs:cluster':
+                    Fn::Sub:
+                      - arn:${AWS::Partition}:ecs:${AWS::Region}:${AWS::AccountId}:cluster/${ClusterName}
+                      - ClusterName:
+                          Fn::ImportValue:
+                            !Sub '${AppName}-${EnvName}-ClusterId'
+            - Sid: SQS
+              Effect: Allow
+              Action:
+                - sqs:GetQueueAttributes
+                - sqs:GetQueueUrl
+              Resource:
+                - !GetAtt EventsQueue.Arn
+                {{- range $topic := .Subscribe.Topics}}
+                {{- if $topic.Queue}}
+                - !GetAtt {{logicalIDSafe $topic.Service}}{{logicalIDSafe $topic.Name}}EventsQueue.Arn
+                {{- end}}
+                {{- end }}
+    ManagedPolicyArns:
+      - !Sub arn:${AWS::Partition}:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole
+
+BacklogPerTaskScheduledRule:
+  Metadata:
+    'aws:copilot:description': "A trigger to invoke the BacklogPerTaskCalculator Lambda function every minute"
+  DependsOn:
+    - BacklogPerTaskCalculatorLogGroup # Ensure log group is created before invoking.
+  Type: AWS::Events::Rule
+  Properties:
+    ScheduleExpression: "rate(1 minute)"
+    State: "ENABLED"
+    Targets:
+      - Arn: !GetAtt BacklogPerTaskCalculatorFunction.Arn
+        Id: "BacklogPerTaskCalculatorFunctionTrigger"
+
+PermissionToInvokeBacklogPerTaskCalculatorLambda:
+  Type: AWS::Lambda::Permission
+  Properties:
+    FunctionName: !Ref BacklogPerTaskCalculatorFunction
+    Action: lambda:InvokeFunction
+    Principal: events.amazonaws.com
+    SourceArn: !GetAtt BacklogPerTaskScheduledRule.Arn
+
+AutoScalingPolicyEventsQueue:
+  Metadata:
+    'aws:copilot:description': "An autoscaling policy to maintain {{.Autoscaling.QueueDelay.AcceptableBacklogPerTask}} messages/task for EventsQueue"
+  Type: AWS::ApplicationAutoScaling::ScalingPolicy
+  Properties:
+    PolicyName: !Join ['-', [!Ref WorkloadName, BacklogPerTask, !GetAtt EventsQueue.QueueName]]
+    PolicyType: TargetTrackingScaling
+    ScalingTargetId: !Ref AutoScalingTarget
+    TargetTrackingScalingPolicyConfiguration:
+      ScaleInCooldown: 120
+      ScaleOutCooldown: 60
+      CustomizedMetricSpecification:
+        Namespace: !Sub '${AppName}-${EnvName}-${WorkloadName}'
+        MetricName: BacklogPerTask
+        Statistic: Average
+        Dimensions:
+          - Name: QueueName
+            Value: !GetAtt EventsQueue.QueueName
+        Unit: Count
+      TargetValue: {{.Autoscaling.QueueDelay.AcceptableBacklogPerTask}}
+
+{{- $acceptableBacklog := .Autoscaling.QueueDelay.AcceptableBacklogPerTask }}
```

Comment:
Can we possibly use `$acceptableBacklog` for all `{{.Autoscaling.QueueDelay.AcceptableBacklogPerTask}}`?

Reply:
Done!